packages/parcel-transformer/src/transformer.tsx

```diff
@@ -37,8 +40,8 @@ export default new Transformer<UserlandOpts>({
     return false;
   },
 
-  async parse({ asset }) {
-    if (!asset.isSource) {
+  async parse({ asset, config }) {
+    if (!asset.isSource && !config.extract) {
```

Comment:
Shouldn't the asset be parsed when extraction is disabled?

Reply:
When the asset is not source and extraction is disabled, the transformer has nothing to do so this should reduce build time